optuna_dashboard/ts/components/CompareStudies.tsx

```diff
@@ -213,14 +213,30 @@ export const CompareStudies: FC<{
                         >
                           {`# ${study.study_id}`}
                         </Typography>
+                      </Box>
+                      <Box
+                        sx={{
+                          display: "flex",
+                          flexDirection: "row",
+                          width: "100%",
+                        }}
+                      >
                         <Chip
                           color="primary"
                           label={
                             study.directions.length === 1
                               ? `${study.directions.length} objective`
                               : `${study.directions.length} objectives`
                           }
-                          sx={{ margin: theme.spacing(0) }}
+                          size="small"
+                          variant="outlined"
+                        />
+                        <span style={{ margin: "4px" }} />
```

Comment:
nit: Can we use `theme.spacing` instead of the magic number?

```suggestion
                        <span style={{ margin: theme.spacing(0.5) }} />
```

Reply:
Fixed.